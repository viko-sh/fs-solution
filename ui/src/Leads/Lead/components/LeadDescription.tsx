import React from 'react';
import { LeadProps } from '../type';
import { Description } from '../style';
export const LeadDescription = ({ description }: Partial<LeadProps>) => {
  return (
    <Description>
      <p>{description}</p>
    </Description>
  );
};
