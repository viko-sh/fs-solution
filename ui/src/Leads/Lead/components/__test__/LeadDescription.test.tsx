import React from 'react';
import { mount } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

import { LeadDescription } from '../LeadDescription';

const defaultProps = {
  description: 'Praesent elit '
};

const mountComponent = (customProps: any = {}) => {
  const props = { ...defaultProps, ...customProps };
  const wrapper = mount(<LeadDescription {...props} />);
  return { props, wrapper };
};

describe('LeadDescription component', () => {
  it('Should render header', done => {
    const { wrapper } = mountComponent();
    expect(wrapper).toHaveLength(1);
    done();
  });
  it('should be match with snapshot', done => {
    // Arrange (no Act required)
    const { wrapper } = mountComponent();
    // Assert
    expect(wrapper).toMatchSnapshot();
    done();
  });
});
