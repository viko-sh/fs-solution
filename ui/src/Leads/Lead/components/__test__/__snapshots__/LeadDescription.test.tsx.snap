// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeadDescription component should be match with snapshot 1`] = `ReactWrapper {}`;
